apps/cookbook/src/app/examples/modal-example/modal-example-alert.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component } from '@angular/core';
+
+import { ModalController, ModalFlavor } from '@kirbydesign/designsystem';
+
+import { ModalEmbeddedAlertExampleComponent } from './alert-example/modal-example-embedded-alert.component';
+
+const config = {
+  selector: 'cookbook-modal-example-alert',
+  template: `<button kirby-button size="lg" (click)="showModal('modal')">Show modal</button>
+<button kirby-button size="lg"(click)="showModal('drawer')">Show drawer</button>`,
+  codeSnippet: `constructor(private myService: MyService) {}
```

Comment:
Consider adding to button text to reflect what will happen:
```suggestion
  template: `<button kirby-button size="lg" (click)="showModal('modal')">Show modal (with alert)</button> 
<button kirby-button size="lg"(click)="showModal('drawer')">Show drawer (with alert)</button>`,
  codeSnippet: `constructor(private myService: MyService) {}
```